### PR TITLE
Improve website

### PR DESCRIPTION
**Pull request by dmigwi**
image

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -3,14 +3,11 @@ title = "Cryptopower"
...
 
 [params]
-
-    favicon = "../images/favicon-32x32.png"
-    main = "Cryptopower"
-    info = "A cross-platform desktop SPV multichain (DCR, BTC & LTC) wallet built with gio."
-
+  favicon = "../images/cryptopower.png"
+  main = "CRYPTOPOWER"
+  info = "A cross-platform SPV (DCR, BTC & LTC) wallet built with privacy in mind."
```
Comment (dreacot) [inline config.toml]:
```suggestion
  info = "A cross-platform SPV (DCR, BTC & LTC) privacy wallet built with go."
```

build with go is our niche it needs to be there

File: layouts/about/list.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <section id="about" class="centered about section">
   <div class="container">
     <h2>About</h2>
-    <h3>Planet Decred is an open-source community where developers from around the world can integrate, develop and build for Decred. We organize to provide the infrastructure, tools, and support to bring new members up to speed quickly. We are a resource to help the entire Decred economy grow. To learn more about the Decred blockchain, visit the <a href="https://docs.decred.org/" target="_blank">documentation</a>. For developers, see the <a href="https://devdocs.decred.org/" target="_target">dev docs.</a>  Come <a href="/chat">chat</a> with us.</h3>
+    <h3>Cryptopower is built by an open-source community of developers from around the world. We organize to provide the infrastructure, tools, and support to bring new members up to speed quickly. We are a resource to help the entire blockchain economy grow. Come <a href="https://matrix.to/#/!lDqGdnRvtobPSBBxGT:decred.org?via=decred.org&via=matrix.org">chat</a> with us.</h3>
```
Comment (dreacot) [inline layouts/about/list.html]: this should link to the cryptopower room, not the general room

File: layouts/chat/list.html [hunk 3]
```diff
@@ -16,7 +16,7 @@ <h3 style="font-weight: 600;">
           Web interface and Sign Up:
         </h3>
         <h3>
-          Use <a href="https://chat.planetdecred.org/" target="_blank">chat.planetdecred.org</a>. For development, please use the same name you use on github. <br>Once logged in, join the Planet Decred room by clicking on <strong>Explore</strong> on the top left. </p>
+          Use <a href="https://matrix.to/#/!lDqGdnRvtobPSBBxGT:decred.org?via=decred.org&via=matrix.org" target="_blank">matrix.org</a>. For development, please use the same name you use on github. <br>Once logged in, join the Planet Decred room by clicking on <strong>Explore</strong> on the top left. </p>
```
Comment (dreacot) [inline layouts/chat/list.html:19]: this should link to the cryptopower room

**Comment by dreacot** [comment]
Screenshot from 2023-06-30 08-35-26

i get an application error at the bottom of the screen

**Comment by dreacot** [comment]
the homepage isn't actually centralized on the screen, compare with the downloads page to see what i mean

| | |
|-|-|

**Review by dreacot — CHANGES_REQUESTED** [review]

**Comment by dmigwi** [comment]
> Screenshot from 2023-06-30 08-35-26
> 
> i get an application error at the bottom of the screen

I can't view this error on my end.

**Review by dreacot — APPROVED** [review]

**Pull request merged**